test/errors-spec.js

```diff
@@ -176,23 +177,23 @@ describe("Parser.expect calls:", function() {
         // Can't test for the [ of an optional group since it's optional
         it("complains about missing } for color", function() {
             expect`\textcolor{#ffffff{text}`.toFailWithParseError(
-                   "Invalid color: '#ffffff{text' at position 12:" +
-                   " \\textcolor{#̲f̲f̲f̲f̲f̲f̲{̲t̲e̲x̲t̲}");
+                   "Unexpected end of input in a macro argument," +
+                   " expected '}' at end of input: …r{#ffffff{text}");
```

Comment:
Removing the position information may make it hard to track down errors in some cases.  The underlining was also helpful because it made it really easy to see the range of the issue.  Looking at some of the test cases below, we still have both of these features in those test cases.  Is it hard to maintain those features in these test cases because how macro expansion has changed?

Reply:
@kevinbarabash In case of macros, arguments should be scanned (fetched) first, so scanning errors are detected before parsing errors, hence changes in error position.

Reply:
I see, this is just how we've been reporting parse for some time now.